sci-libs/gsl/files/gsl-drop-broken-rng-tests.patch

```diff
@@ -0,0 +1,34 @@
+Commit: https://github.com/Jamim/gsl/commit/96484895
+Author: Aliaksei Urbanski <aliaksei.urbanski@gmail.com>
+Date:   Sun Jul 14 20:09:09 2024 +0300
+
+    ✅ Drop broken rng tests
```

Comment:
Hi, no, sorry. Don't do that.

Reply:
I would've allowed it if it were 🎲 ...

Reply:
Also, emoji aside, what is broken about them? Is there an upstream bug link describing the brokenness?